Gemfile

```diff
@@ -0,0 +1,17 @@
+source ENV['GEM_SOURCE'] || "https://rubygems.org"
+
+def vanagon_location_for(place)
+  if place =~ /^(git[:@][^#]*)#(.*)/
+    [{ :git => $1, :branch => $2, :require => false }]
+  elsif place =~ /^file:\/\/(.*)/
+    ['>= 0', { :path => File.expand_path($1), :require => false }]
+  else
+    [place, { :require => false }]
+  end
+end
+
+gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || '0.14.1')
+gem 'packaging', '~> 0.6', :git => 'https://github.com/puppetlabs/packaging.git'
+gem 'rake', '~> 12.0'
+
+#gem 'rubocop', "~> 0.34.2"
```

Comment:
Is there a reason this is still here, but commented out?

Reply:
I copied the gemfile from pdk, that's what they had https://github.com/puppetlabs/pdk-vanagon/blob/master/Gemfile#L17

Reply:
ookay!